inspectkv/inspectkv.go

```diff
@@ -68,23 +72,47 @@ func nextIndexVals(data []interface{}) []interface{} {
 	return append(data, nil)
 }
 
-// IndexRow is the index information composed of a handle and values.
-type IndexRow struct {
+// RecordData is the record data composed of a handle and values.
+type RecordData struct {
 	Handle int64
 	Values []interface{}
 }
 
-// ScanIndexData scans the index handles and values in a limited number.
-// It returns data and the next startVals.
+// GetIndexRecordsCount returns the total number of the index records.
+func GetIndexRecordsCount(txn kv.Transaction, kvIndex kv.Index, startVals []interface{}) (int64, error) {
```

Comment:
I guess this function returns the whole records count, so need startVals, is it nil already?